doc/monitoring/metrics_reference.rst

```diff
@@ -94,3 +94,39 @@ Those metrics provide specific information about each individual space in Tarant
 * ``tnt_space_count`` - the total tuples count for vinyl.
   This metric always has labels - ``{name="test", engine="vinyl"}``. ``name`` - the name of the space.
   ``engine`` - is the engine of the space.
+
+.. _network:
+
+-------------------------------------------------------------------------------
+ Network
+-------------------------------------------------------------------------------
+
+Network activity stats. This can be used to monitor the load on network, usage peaks and traffic drops.
+
+Sent bytes:
+
+* ``tnt_net_sent_total`` - bytes sent from this instance over network since instance start
+
+* ``tnt_net_sent_rps`` - average number of bytes sent per second in the last 5 seconds
```

Comment:
I suggest not documenting _rps metrics. it lacks clarity and could be easily derived from _total